### DISCUSSION TITLE
Enable edit capabilities to upgrade inspector as Live Editor

### DISCUSSION THREAD

**Discussion by kitepad** [c-smile/sciter-js-sdk#108] [Ideas]
Currently, inspector is read only for opened files (html, css, js). 
It's great help for sciter developer if support live editing or editing with manual reloading.

**Comment by kitepad** [comment]
Or split usciter layout as preview and source. I can edit source file, then reload file in usciter to verify my UI.

**Reply by c-smile** [reply]
usciter supports live reload. So you can edit your files in your IDE/editor and it will reload it.

Is this what you want?

**Reply by kitepad** [reply]
Now, editor+usciter are my development env. But it is NOT my want, because I have to arrange editor window with usciter window in many windows of my laptop. I hope usciter provide source/preview layout like markdown sample app. One window is editor, another is preview.
 
![image](https://user-images.githubusercontent.com/28701482/121621549-efb21100-ca9e-11eb-93f7-5ac1a6acadf2.png)

**Reply by kitepad** [reply]
This is my html/css  editor layout using codepen. It's my basic expectation : 
1. split sciterjs html file as source/runtime.
2. After I edit and save my source, runtime view auto reload it.
3. Source editor can split more to view separate html/js/css.  
4. It's better if have syntax highlight/auto complete/code snippets.  
![image](https://user-images.githubusercontent.com/28701482/121764384-feb7c280-cb75-11eb-9062-df4aad8e8108.png)

**Reply by kitepad** [reply]
I'm trying to modify markdown sample to produce my prototype.

**Reply by kitepad** [reply]
This is my first prototype base on mdview sample. Thanks for Andrew's excellent work and help.
![image](https://user-images.githubusercontent.com/28701482/121850768-d8358b00-cd1f-11eb-830f-b8dc64fa6196.png)
